app/frontend/stylesheets/application.scss

```diff
@@ -1,5 +1,11 @@
 // Config files
+
+@import "config/fonts";
+@import "config/colors";
+
 @import "config/variables";
+
+
 @import "config/setup";
```

Comment:
You can remove everything you added here after deleting the fonts and colors file like I asked.
There should only be the variables and setup import in the `// Config File` section of this file after that

Reply:
Ok